### PR TITLE
Update from facebook

### PR DESCRIPTION
**Pull request by bddppq**

### CODE REVIEW

File: aten/src/TH/THGeneral.h [hunk 1]
```diff
@@ -0,0 +1,174 @@
+#ifndef TH_GENERAL_INC
```
Comment (soumith) [inline aten/src/TH/THGeneral.h]: this file needs to be removed

Comment (soumith) [reply aten/src/TH/THGeneral.h]: so should aten/src/ATen/Declarations.yaml (it's generated)

File: aten/src/THC/THCGeneral.h [hunk 2]
```diff
@@ -0,0 +1,221 @@
+#ifndef THC_GENERAL_INC
```
Comment (soumith) [inline aten/src/THC/THCGeneral.h]: this file should be removed (it's autogenerated)

File: test/test_torch.py [hunk 3]
```diff
@@ -5836,7 +5836,14 @@ def import_module(name, filename):
             return module
 
         with filecontext_lambda() as checkpoint:
+<<<<<<< HEAD
```
Comment (soumith) [inline test/test_torch.py]: these changes are local to fbcode and shouldn't be synced back

File: tools/setup_helpers/generate_code.py [hunk 4]
```diff
@@ -84,10 +84,17 @@ def generate_code(ninja_global=None):
         if not os.path.exists(d):
             os.mkdir(d)
     gen_autograd(
+<<<<<<< HEAD
```
Comment (soumith) [inline tools/setup_helpers/generate_code.py]: these changes are local to fbcode and shouldn't be synced back

Comment (bddppq) [reply tools/setup_helpers/generate_code.py]: @orionr

**Pull request closed without merging**